lib/configuration.js

```diff
@@ -31,7 +31,8 @@ var ALLOWED_CLIENT_SIDE_KEYS = [
   'useSmtp',
   'useffmpeg',
   'isProduction',
-  'maxLoginAttempts'
+  'maxLoginAttempts',
+  'ckEditorAllowedContent'
```

Comment:
`ckEditorAllowedContent` is misleading as there is a [corresponding option](https://docs.ckeditor.com/ckeditor4/latest/api/CKEDITOR_config.html#cfg-allowedContent) in CKEditor.